src/index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="css/vote-CSS.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap'"
+    />
+  </head>
+  <body>
+    <div class="test_box">
+      <div class="u-layout">
+        <img class="u-layout" src="c:\Pics\logo.png" alt="logo" />
```

Comment:
These `png`'s need moving to be local to the folder structure.

Usually in a folder like `src/images`